packages/tasks/package.json

```diff
@@ -24,10 +24,11 @@
 		"format": "prettier --write .",
 		"format:check": "prettier --check .",
 		"prepublishOnly": "pnpm run build",
-		"build": "tsup src/index.ts --format cjs,esm --clean --dts && pnpm run inference-codegen",
+		"build": "tsup src/index.ts --format cjs,esm --clean --dts && pnpm run inference-tgi-import && pnpm run inference-codegen",
```

Comment:
Let's not run `inference-tgi-import` automatically.

It fetches a remote file, so let's run it once a while or in a cron job (to make a PR).

Fetching a remote file like this during a script that runs on `pnpm install` has some security implications, among other things

Reply:
Thanks for the review! Reverted that in https://github.com/huggingface/huggingface.js/pull/629/commits/092a7aa482c4f6e2a9a61596243692872b89e53e. About cron job/automated PRs, that's the plan yes but will be done in a future PR.